dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -459,17 +467,17 @@ public boolean checkUserExists(@ApiParam(hidden = true) @Parameter(hidden = true
     public List<Tool> refreshToolsByOrganization(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
             @ApiParam(value = "User ID", required = true) @PathParam("userId") Long userId,
             @ApiParam(value = "Organization", required = true) @PathParam("organization") String organization,
-            @ApiParam(value = "Docker registry") @QueryParam("dockerRegistry") String dockerRegistry) {
+            @ApiParam(value = "Docker registry", required = true) @QueryParam("dockerRegistry") String dockerRegistry) {
 
         checkUser(authUser, userId);
 
         // Check if the user has tokens for the organization they're refreshing
         checkToolTokens(authUser, userId, organization);
-        if (dockerRegistry != null) {
-            dockerRepoResource.refreshToolsForUser(userId, organization, dockerRegistry);
-        } else {
-            dockerRepoResource.refreshToolsForUser(userId, organization);
+        if (dockerRegistry == null) {
+            throw new CustomWebApplicationException("a repository is now required", HttpStatus.SC_BAD_REQUEST);
```

Comment:
I would just say `A repository is required`.

1. We usually start with capitals (although not always, so we're not that consistent).
2. `now` will become old and I doubt anybody is relying on the current behavior.